Dockerfile.deps

```diff
@@ -89,7 +89,7 @@ RUN CUDA_VERSION=9.0 && \
     rm -f cuda_${CUDA_BUILD}_linux-run
 
 # NVJPEG
-RUN NVJPEG_VERSION=9.0 && \
+RUN NVJPEG_VERSION=9.0-v0.1.2 && \
     NVJPEG_BUILD=9.0.462 && \
     curl -L https://developer.download.nvidia.com/compute/redist/libnvjpeg/cuda-linux64-nvjpeg-${NVJPEG_VERSION}.tar.gz | tar -xzf - && \
```

Comment:
NVJPEG _BUILD is not changing?

Reply:
It should change, yes.  Or just delete that line.